grpc/nodejs/BUILD

```diff
@@ -16,21 +16,50 @@
 #
 
 package(default_visibility = ["//visibility:public"])
-load("@stackb_rules_proto//node:node_grpc_compile.bzl", "node_grpc_compile")
 load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+load("@stackb_rules_proto//node:node_grpc_compile.bzl", "node_grpc_compile")
+load("@npm//@bazel/typescript:index.bzl", "ts_project")
+
 
-node_grpc_compile(
+genrule(
     name = "protocol",
-    deps = [
-        "//protobuf:answer-proto",
-        "//protobuf:concept-proto",
-        "//protobuf:database-proto",
-        "//protobuf:grakn-proto",
-        "//protobuf:options-proto",
-        "//protobuf:query-proto",
-        "//protobuf:session-proto",
-        "//protobuf:transaction-proto",
-    ]
+    outs = [
+        "protobuf/answer_pb.d.ts",
+        "protobuf/answer_pb.js",
+        "protobuf/concept_pb.d.ts",
+        "protobuf/concept_pb.js",
+        "protobuf/database_pb.d.ts",
+        "protobuf/database_pb.js",
+        "protobuf/grakn_pb.d.ts",
+        "protobuf/grakn_pb.js",
+        "protobuf/grakn_grpc_pb.js",
+        "protobuf/grakn_grpc_pb.d.ts",
+        "protobuf/options_pb.d.ts",
+        "protobuf/options_pb.js",
+        "protobuf/query_pb.d.ts",
+        "protobuf/query_pb.js",
+        "protobuf/session_pb.d.ts",
+        "protobuf/session_pb.js",
+        "protobuf/transaction_pb.d.ts",
+        "protobuf/transaction_pb.js",
```

Comment:
Set of outputs changed to reflect we a) now only have a single service and b) we now have both a .js and a .d.ts file for every incoming proto file.